typings/index.d.ts

```diff
@@ -218,7 +218,7 @@ declare module 'discord.js' {
     public fetchVoiceRegions(): Promise<Collection<string, VoiceRegion>>;
     public fetchWebhook(id: Snowflake, token?: string): Promise<Webhook>;
     public generateInvite(options?: InviteGenerationOptions | PermissionResolvable): Promise<string>;
-    public login(token?: string): Promise<string>;
+    public login(token?: string, sessionID?: string|string[]): Promise<string>;
```

Comment:
Wouldn't you just pass one sessionID instead of an array of session IDs to login?

Reply:
From what I understand, each shard has it's own session. So passing an array allows you to assign each of your shards with a session ID